src/components/periodic-report/periodic-report.service.ts

```diff
@@ -198,33 +241,15 @@ export class PeriodicReportService {
       type: props.type,
       start: props.start,
       end: props.end,
-      reportFile: await this.files.resolveDefinedFile(
-        {
-          value: result.reportFileId,
-          canEdit: true,
-          canRead: true,
-        },
-        session
-      ),
+      reportFile: {
+        value: result.reportFileId,
+        canEdit: true,
+        canRead: true,
+      },
```

Comment:
Pass `reportFileId` as `reportFile` to `secureProperties()` so that authorization can be properly applied